### ISSUE TITLE
Suppressing GPU Fact info when using MAGMA

### ISSUE THREAD

**Issue by abhyshr** [LLNL/hiop#50]
When running HiOp with MAGMA, the following output is being printed to stdout.
```
GPU FACT in 0.0493771 sec at TFlops: 0.000712128
GPU FACT in 0.0434769 sec at TFlops: 0.00080877
GPU FACT in 0.0436565 sec at TFlops: 0.000805443
GPU FACT in 0.043938 sec at TFlops: 0.000800282
GPU FACT in 0.0430558 sec at TFlops: 0.00081668
GPU FACT in 0.0432639 sec at TFlops: 0.000812752
GPU FACT in 0.0433745 sec at TFlops: 0.00081068
GPU FACT in 0.0432896 sec at TFlops: 0.000812269
GPU FACT in 0.0591214 sec at TFlops: 0.000594756
GPU FACT in 0.0609099 sec at TFlops: 0.000577292
```
I'd like this output to be suppressed and perhaps be allowed based on the verbose level. 

I think it is printed via [this line](https://github.com/LLNL/hiop/blob/7e8836c2de665616b0e8438ac365072965be257d/src/LinAlg/hiopLinSolverIndefDenseMagma.hpp#L92)

**Comment by cnpetra** [comment]
noted. gpu-devperf branch devs will take care of this. it is safe to suppressed it till the PR will be approved

**Comment by cnpetra** [comment]
addressed by PR #51 

note that additional performance profiling and reporting in the linear and kkt solves can be enabled using the string option
```
time_kkt on
```
